llvm/lib/Target/AArch64/AArch64GlobalISelPatterns.td

```diff
@@ -0,0 +1,36 @@
+//===-- AArch64GlobalISelPatterns.td - GlobalISel patterns -*- tablegen -*-===//
```

Comment:
I don't know that much about GlobalISel, but I thought we could reuse a lot of the existing patterns. Do we really need to duplicate all this functionality?

Reply:
I don't' want to duplicate patterns. We re-use a lot of the existing ones. There are just some patterns missing, e.g., fneg and fabs.

Reply:
Patterns like ,e.g, extract_subvector work great for use. We have issues with predicated instructions. I want to add patterns that unpredicate instructions.